packages/contracts-bedrock/scripts/Deploy.s.sol

```diff
@@ -9,6 +9,8 @@ import { stdJson } from "forge-std/StdJson.sol";
 import { Deployer } from "./Deployer.sol";
 import { DeployConfig } from "./DeployConfig.s.sol";
 
+import { Safe } from "safe-contracts/Safe.sol";
+import { SafeProxyFactory } from "safe-contracts/proxies/SafeProxyFactory.sol";
```

Comment:
Any reason to have both but only build 1?

Reply:
Not really. I've added it.